radar/tools/bruno.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Bruno"
+ring: assess
+quadrant: "tools"
+tags: [api, web, mobile]
+---
+
+[Bruno](https://www.usebruno.com/) is an open-source API client that's gaining traction as a lightweight alternative to Postman. What sets it apart is its "collections as code" philosophy - API collections are stored in plain text files that can be version controlled, making it a natural fit for developer workflows. The tool offers a fast, native desktop experience with a clean interface and features like environment variables, request chaining, and scripting support. While it's relatively new and the ecosystem is still growing, Bruno's git-friendly approach and focus on developer experience make it worth evaluating, especially for teams already embracing Infrastructure as Code principles.
```

Comment:
can we add "Use cases" section and "Reference of usage in our organization"? ie. https://github.com/search?q=org%3Apagopa+bruno.json&type=code

Reply:
I took my cue from the other tools like maven and they don't have the use case part. I've meaning it as optional.  However, I am in favor of it. Go ahead